dask_saturn/core.py

```diff
@@ -126,6 +126,13 @@ def __init__(
                 "have a different dask-saturn version on your dask cluster."
             )
 
+    # pylint: disable=unused-argument
+    def __await__(self):
```

Comment:
did `pylint` really complain about `self` being unused in this method? That's surprising to me.

Reply:
I don't think so - I put it in because of https://github.com/saturncloud/dask-saturn/pull/51#discussion_r551516831.  I don't really know pylint best practices, but I can take it out

Reply:
if it isn't necessary, I recommend taking it out. The fewer `#pylint: disable`s, the better. If we added another parameter to this method in the future and then misspelled it or something, this disable comment would prevent pylint from catching that

Reply:
oh yeah sorry, I thought it was needed.